### PR TITLE
Js us40 41

### PR DESCRIPTION
**Pull request by jrsewell400**
Made the following changes:
- Add feature tests to check that admin can enable merchant and see a flash message as well as active items on "/merchants/:id/items"
- Added model test for activate_items instance method
- Updated Merchant model test to add additional tests for enabling and disabling merchant
- Added enable merchant link to admin dashboard
- Added another storefront and items to seed file
- 99.8% test coverage, all tests passing

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@ def update
       merchant.deactivate_items
       flash[:notice] = "#{merchant.name} is now disabled."
       redirect_to admin_merchants_path
+    else  
+      merchant.update(enabled?: true)
+      merchant.activate_items
+      flash[:notice] = "#{merchant.name} is now enabled."
+      redirect_to admin_merchants_path
```
Comment (steve-meyers) [inline app/controllers/admin/merchants_controller.rb:21]: This code looks great and really clean!

File: app/views/admin/merchants/index.html.erb [hunk 2]
```diff
@@ -22,7 +22,9 @@
         </td>
         <td>
           <% if merchant.enabled? %>
-            <%= link_to 'Disable Merchant', admin_merchant_path(merchant.id), method: :patch %>
+            <%= button_to 'Disable Merchant', admin_merchant_path(merchant.id), method: :patch %>
+          <% else %>
+            <%= button_to 'Enable Merchant', admin_merchant_path(merchant.id), method: :patch %>
```
Comment (steve-meyers) [inline app/views/admin/merchants/index.html.erb:27]: Nice! I was going to suggest that this be a button instead and you read my mind! 🧠

File: spec/models/merchant_spec.rb [hunk 3]
```diff
@@ -76,10 +76,27 @@
...
+      pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", active?: true, inventory: 32)
+      dog_bone = dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?: true, inventory: 21)
+      dog_bowl = dog_shop.items.create(name: "Dog Bowl", description: "Great dog bowl!", price: 7, image: "https://www.talltailsdog.com/pub/media/catalog/product/cache/a0f79b354624f8eb0e90cc12a21406d2/u/n/untitled-6.jpg", active?: true, inventory: 32)
+
+      dog_shop.activate_items
+      
+      expect(pull_toy.active?).to eq(true)
+      expect(dog_bone.active?).to eq(true)
+      expect(dog_bowl.active?).to eq(true)
     end
```
Comment (steve-meyers) [inline spec/models/merchant_spec.rb:100]: Nice these tests look great! 🏅

**Review by steve-meyers — APPROVED** [review]
Looks good! I like the updated model testing. 🧪

File: app/controllers/admin/merchants_controller.rb (hunk 1 above)
Comment (zachholcomb) [reply app/controllers/admin/merchants_controller.rb:21]: looks great maybe helper method for a lot of that if logic, to shorten the controller action and follow single responsibility

**Comment by zachholcomb** [comment]
Looks great jordan!

**Pull request merged**